reactor-core/src/main/java/reactor/core/scheduler/SingleScheduler.java

```diff
@@ -78,6 +81,25 @@ public boolean isDisposed() {
 		return current != null && current.currentResource == TERMINATED;
 	}
 
+	@Override
+	public void init() {
+		SchedulerState<ScheduledExecutorService> b = SchedulerState.init(
```

Comment:
Same as for previous cases. Test and Test-and-set pattern should be used to avoid redundant allocations